pkgs/development/misc/resholve/default.nix

```diff
@@ -28,7 +28,7 @@ let
     stripTests = true;
     enableOptimizations = false;
   };
-  callPackage = lib.callPackageWith (pkgs // { python27 = python27'; });
+  callPackage = lib.callPackageWith (pkgsBuildHost // { python27 = python27'; });
```

Comment:
This seems to be the key change to support cross compilation. Eager to have this well-vetted, though :)

@thiagokokada @k900 Does this smell right to you?